ansible/playbook.yml

```diff
@@ -14,6 +14,9 @@
 
   # Abort if a lockfile is present; otherwise create the file.
   pre_tasks:
+    - name: Get ssl certs
+      import_tasks: "tasks/lets_encrypt.yml"
+
```

Comment:
You will want to move this to after the lockfile check (which comes directly below this step).

Reply:
I was having trouble with that. I was getting yaml/ansible syntax errors when I put it after the lockfile. I think that since this piece isn't blocked by another process it should be fine here for now. If we start having issues we can look to move it, maybe?

Reply:
What errors are you getting?

Reply:
It must have just been a user error. I moved it in f99f4e9 and it's looking like it worked